README.md

```diff
@@ -0,0 +1,69 @@
+# OCP DNSNameResolver
+
+## Name
+
+OCP DNSNameResolver - updates the status of the [DNSNameResolver](https://github.com/openshift/api/blob/master/network/v1alpha1/types_dnsnameresolver.go)
+custom resources with IP addresses of matching resolved DNS names.
+
+## Description
+
+The spec of DNSNameResolver custom resource takes as input a DNS name. The DNS name can be either a regular or a wildcard DNS name. The plugin intercepts
+the DNS lookups of the matching DNS names and updates the status of these CRs with the IP addresses of the matching resolved names.
```

Comment:
`matching` what? Did you mean that the plugin intercepts the DNS lookups for the DNS name specified in `DNSNameResolver`?
`these CRs`? There is another CR apart from `DNSNameResolver`?

Reply:
> `matching` what? Did you mean that the plugin intercepts the DNS lookups for the DNS name specified in `DNSNameResolver`?

Mostly yes. For a `DNSNameResolver` CR of a wildcard DNS name, it'll intercept those DNS names which match the wildcard DNS name.

> these CRs? There is another CR apart from DNSNameResolver?

A regular DNS name can match 2 possible `DNSNameResolver` CRs, the `DNSNameResolver` CR for the regular DNS and the `DNSNameResolver` CR for the corresponding wildcard DNS name. The status of both the CRs should be updated.